PubSubSpec.md

```diff
@@ -92,6 +92,17 @@ The following well-known additional blocks are RECOMMENDED.
 Timestamp = TimestampNameComponent
 ```
 
+### Name Mapping Delivery Optimization
+
+To optimize the delivery of mapping data, SVS-PS implementations MAY piggyback mapping data in the Sync Interest's Application Parameters. If present, the mapping data SHOULD be appended as a single block after the StateVector in the Application Parameters. The mapping data MUST be encoded as a TLV block of type `MappingData` as defined above.
```

Comment:
I suggest writing `ApplicationParameters` (with backticks and no space) to refer to the TLV, instead of "Application Parameters"